include/tcp_socket.h

```diff
@@ -0,0 +1,94 @@
+/***************************************************************************//**
```

Comment:
A short description would help in the commit body. For example what do the files added do and how do they relate to each other. Maybe how they help in building a WiFi connection.

Reply:
done